Sources/SwifterSwift/Foundation/NSAttributedStringExtensions.swift

```diff
@@ -134,6 +134,26 @@ public extension NSAttributedString {
     }
 }
 
+// MARK: - Initializers
+
+public extension NSAttributedString {
+    #if os(iOS) || os(tvOS)
+    /// SwifterSwift: Creates an attributed string with an image centered by a font.
+    /// - Parameters:
+    ///   - image: The image to create attributed string with (the image will not be scaled).
+    ///   - font: The font by which the image should be centered.
+    @available(iOS 7.0, tvOS 9.0, *)
```

Comment:
You can remove this line, since the minimum versions for this project already allows it.